src/Sarif.Converters/CppCheckConverter.cs

```diff
@@ -20,6 +20,8 @@ public CppCheckConverter()
             _strings = new CppCheckStrings(_nameTable);
         }
 
+        public override string ToolName => ToolFormat.CppCheck;
```

Comment:
>CppCheck [](start = 54, length = 8)

I know we discussed, and I think we agreed, not to assume that the `ToolFormat` would also make a good tool name. In most of the other converters, you're just hard-coding a string here (`"CppCheck"`).

Reply:
in fact, most converters still use ToolFormat. I'm not going to invest more here, I spent hours on this change DRYing out test code, removing embedded literals. We can revisit this in the future if need be.

---
In reply to: [256111009](https://github.com/Microsoft/sarif-sdk/pull/1277#discussion_r256111009) [](ancestors = 256111009)